src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -57,11 +57,7 @@
 import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
 import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
 import org.glassfish.hk2.api.MultiException;
```

Comment:
yeah that was the first thing I thought of but thought it might be a bit presumptuous to update the jdk version in this PR :)